assets/js/src/core/modules/asset/editor/shared-tab-manager/tabs/custom-metadata/custom-metadata-container.tsx

```diff
@@ -150,6 +148,7 @@ export const CustomMetadataTabContainer = (): React.JSX.Element => {
                 />
 
                 <Select
+                  className={ styles.customMetadataSelect }
```

Comment:
Would it also be possible to use our utility classes for this case like "w-full"`?
That way we would be able to delete the related style file completely. :)